lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
+
+util.inherits(VOLAPI, RestifyClient);
+
+
+VOLAPI.prototype.close = function close() {
+    this.client.close();
+};
+
+VOLAPI.prototype.createVolume =
+    function createVolume(params, options, callback) {
+        if (typeof (options) === 'function') {
+            callback = options;
```

Comment:
I'm not a fan of this double-indentation style just because the function name and/or args was long. That width for just the start of the method should impact indentation for all the lines of the function,IMHO.